cmd/storage.go

```diff
@@ -52,16 +52,22 @@ var storageCreateCmd = &cobra.Command{
 }
 
 var storageUnmountCmd = &cobra.Command{
-	Use:   "unmount [storage name]",
-	Short: "Unmount storage from the current component",
-	Long: `Unmount storage from the current component.
+	Use:   "unmount [storage/path name]",
```

Comment:
we can update this based on https://github.com/redhat-developer/odo/issues/516
`"PATH | STORAGE_NAME"`